DESCRIPTION

```diff
@@ -121,7 +121,11 @@ Authors@R:
       person(given = "Christoph",
              family = "Molnar",
              role = "ctb",
-             email = "christoph.molnar@stat.uni-muenchen.de"))
+             email = "christoph.molnar@stat.uni-muenchen.de"),
+			person(given = "Annette",
```

Comment:
Just a general comment: It is preferred to use whitespace instead of tabs in most coding areas. You're still often using tabs. I'll clean this myself here. 

But please really make sure to stick with the alignment rules and "whitespace only" in mlr in all cases. This prevents me from cleaning up and commenting on it each time :)

Reply:
Sorry! Thanks.